test/continuous.jl

```diff
@@ -74,3 +74,8 @@ end
 #     println("    testing $(distr)")
 #     test_samples(distr, n_tsamples)
 # end
+
+# Test for non-Float64 input
+using DualNumbers
+@test string(logpdf(Normal(0,1),big(1))) == "-1.418938533204672741780329736405617639861397473637783412817151540482765695927251"
```

Comment:
Does that test pass? Equality might be too strict a requirement if the exact order of operations isn't consistent.